### ISSUE TITLE
Conversion failures are not visible anywhere in the standalone tool

### ISSUE THREAD

**Issue by moradin** [lotsopa/Savvy#12]
There should be an output window on the bottom

**Comment by lotsopa** [comment]
You are correct, there are no specific error codes in the standalone tool currently. A proper output window is in the works. It should be implemented in the next release.
